### DISCUSSION TITLE
Shared Environment Variables: Link all existing variables to Project

### DISCUSSION THREAD

**Discussion by TSIA-SN** [vercel/community#1129] [Idea]
Currently if you want to link existing Shared Environment Variables to a new project (or just a different existing project that you didn't link them to initially when creating them), from what I can tell, you have to click Edit for each variable and then link it to the project. If you have a lengthy list and want to link all Environment Variables, this could be subject to mistakes as you might potentially miss a variable while scrolling through the list. It would be nice if there would be a checkbox beside the each variable (with a "Select/Deselect All" option at the top) and then a dropdown to select which project(s) the selected variables should be linked to.

Alternatively, there could just be an option to select a project and have all environment variables be linked to it.

**Comment by amyegan** [comment]
Thanks for sharing this idea @TSIA-SN! I passed it along to the team 😁
